genesyscloud/telephony_providers_edges_phonebasesettings/resource_genesyscloud_telephony_providers_edges_phonebasesettings_test.go

```diff
@@ -23,6 +23,8 @@ func TestAccResourcePhoneBaseSettings(t *testing.T) {
 		description1                   = "test description 1"
 		description2                   = "test description 2"
 		phoneMetaBaseId                = "generic_sip.json"
+		stationPersistTimeout          = "2000"
+		stationPersistTimeoutUpdate    = "3000"
```

Comment:
With `line_meta_base_id` as "generic_sip.json" , I noticed that the bool and int fields do not appear in the Admin UI under General for a phone base setting. But they do for "inin_webrtc_softphone.json". It might be worth investigating it these new fields are only relevant for a particular phone_meta_base_id type

Reply:
We see the attributes under the general tab for generic_sip, also there are several types of telephony that aren't seen in hollywoo, so it may not be easy to add a validation